### ISSUE TITLE
ARTILLERY_CLOUD_API_KEY has no effect in .env file

### ISSUE THREAD

**Issue by bessey** [artilleryio/artillery#2582]
ARTILLERY_CLOUD_API_KEY has to be in your actual environment, whereas .env works for other environment variables.

Version info: 
```

        ___         __  _ ____
  _____/   |  _____/ /_(_) / /__  _______  __ ___
 /____/ /| | / ___/ __/ / / / _ \/ ___/ / / /____/
/____/ ___ |/ /  / /_/ / / /  __/ /  / /_/ /____/
    /_/  |_/_/   \__/_/_/_/\___/_/   \__  /
                                    /____/


VERSION INFO:

Artillery: 2.0.7
Node.js:   v18.19.0
OS:        darwin
```

Running this command: 
```
$ npx artillery run-fargate --dotenv ./.env  --count 1 --record test.yml
```

I expected to see this happen:

The test runs with results pushed to Artillery Cloud, because my Artillery Cloud key is set in the .env file

Instead, this happened:

Error: API key is required to record test results to Artillery Cloud
See https://docs.art/get-started-cloud for more information

Files being used:
```
# .env
ARTILLERY_CLOUD_API_KEY=<redacted>
```

If you run with a true env var, i.e. `ARTILLERY_CLOUD_API_KEY npx artillery run-fargate` it works fine, but this makes it unique in this regard I think?

**Comment by hassy** [comment]
thanks for the bug report @bessey, will fix in the upcoming v2.0.8 release (due on Monday)

**Comment by bernardobridge** [comment]
Now released on [v.2.0.8](https://github.com/artilleryio/artillery/releases/tag/artillery-2.0.8)
